velox/common/base/SpillConfig.h

```diff
@@ -69,8 +69,8 @@ struct SpillConfig {
   /// the next level of recursive spilling.
   int32_t spillLevel(uint8_t startBitOffset) const;
 
-  /// Checks if the given 'startBitOffset' and 'numPartitionBits' has exceeded
-  /// the max hash join spill limit.
+  /// Checks if the given 'startBitOffset' has exceeded the max hash join spill
```

Comment:
Why is this comment referring to hash join? Is this a typo?

Reply:
Fixed.